theme_anelusia/views/snippets/s_company_team_basic.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_company_team_basic" inherit_id="website.s_company_team_basic">
+    <!-- Team #01 -->
```

Comment:
The structure is a bit weird, shouldn't this change be under `<!-- Team #01 - Img -->` (same for other team members)